src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java

```diff
@@ -1,6 +1,6 @@
 /*
  The MIT License (MIT)
- Copyright (c) 2016 Martin Reinhardt
+ Copyright (c) 2018 Raven
```

Comment:
You cannot remove the original copyright owner from this code, you can only add your copyright claim for your own additions/modifications:

```
Original work Copyright (c) 2016 Martin Reinhardt
Modified work Copyright (c) 2018 Raven
```

Reply:
👍Done, Was not aware of this.